server/router/submission.route.js

```diff
@@ -0,0 +1,105 @@
+var express = require('express');
+const router = express.Router();
+
+const Submission = require('../models/submission.model');
+
+
+/**
+ * @swagger
+ * /api/submissions:
+ *  get:
+ *    summary: Retrieve a list of all of the submissions
+ *    produces:
+ *       - application/json
+ *    tags:
+ *      - submissions
+ */
+ router.get('/', async ({ res }) => {
+    res.status(200).send(await Submission.find({}));
+  });
+
+/**
+ * @swagger
+ * /api/submissions/{teamID}/{submissionID}:
+ *   get:
+ *     summary: Retrieves the specified submission from the specified team
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - in: path
+ *         name: submissionID
+ *         required: true
+ *         type: string
+ *         description: The submission ID
+ *     tags:
+ *       - submissions
+ */
+router.get('/:submissionID', async (req, res) => {
```

Comment:
For example,
You can just leave the code:
router.get('/:submissionId', /* call function to get submissionID here */